tidy3d/plugins/design/result.py

```diff
@@ -61,23 +62,24 @@ class Result(Tidy3dBaseModel):
         description="Source code for the function evaluated in the parameter sweep.",
     )
 
-    task_ids: Tuple[Tuple[str, ...], ...] = pd.Field(
+    task_ids: Tuple[str, ...] = pd.Field(
         None,
         title="Task IDs",
         description="Task IDs for the simulation run in each data point. Only available if "
         "the parameter sweep function is split into pre and post processing and run with "
-        "'Design.run_batch()', otherwise is ``None``. "
-        "To access all of the data, see ``batch_data``.",
+        "'Design.run_batch()', otherwise is ``None``. ",
```

Comment:
this syntax changed, so maybe "`Design.run()` with `fn_post` defined"? not sure.